setup.py

```diff
@@ -4,11 +4,11 @@
 from setuptools import find_packages, setup
 
 if sys.version_info < (3, 8, 0):
-    raise OSError(f'OpenGPT requires Python >=3.8, but yours is {sys.version}')
+    raise OSError(f'RunGPT requires Python >=3.8, but yours is {sys.version}')
 
 try:
-    pkg_name = 'open-gpt-torch'
-    libinfo_py = path.join(path.dirname(__file__), 'open_gpt', '__init__.py')
+    pkg_name = 'run-gpt-torch'
```

Comment:
maybe we can use `run-gpt` instead of `run-gpt-torch`

Reply:
@huangkaipeng4399 yes, the suffix `-torch` is only for passing the name validation of pypi. We can directly use `run-gpt` as the Python package name.